### DISCUSSION TITLE
Is It possible to completely replace default Windows 10 volume bar with ModerFlayouts?

### DISCUSSION THREAD

**Discussion by rakitind00** [ShankarBUS/ModernFlyouts#1289] [Q&A]
I really like this app, and would love to have it by default when changing the volume with my mouse , not only the fn keys ( Win 10). 
![image](https://github.com/ModernFlyouts-Community/ModernFlyouts/assets/83398802/acd69c5e-f63f-429b-bd3e-56dcce494357)

**Comment by prakharb5** [comment]
When changing volume by scrolling from the tray icon, you get a popup anyway. I dont believe there is a need to show the flyout
